eva_submission/eload_validation.py

```diff
@@ -110,11 +111,14 @@ def detect_and_optionally_merge(self, merge_per_analysis):
                 self.error('Unsupported merge type!')
 
         if merge_per_analysis:
+            if not validate_aliases(vcfs_by_analysis.keys()):
+                self.error('Analysis aliases not valid as unique merged filenames, will not merge.')
+                return
```

Comment:
I think this should be reported in the `_vcf_merge_report` so it can be actioned.